### ISSUE TITLE
request: recursive install

### ISSUE THREAD

**Issue by bendichter** [mobeets/mpm#58]
When I say `mpm install buzcode` I want the `requirements.txt` file in `buzcode` to be read and those packages to be installed, and then for those packages I want the `requirements.txt` file to be read and installed...

Do you think this is something `mpm` could do?

**Comment by mobeets** [comment]
Yeah I think something like this could be done. This is probably a sort of pandora's box (actually trying to manage installations), but definitely possible.

**Comment by philchiu-envision** [comment]
I'm interested in this as well. Slightly related -- you might need some sort of dependency resolution (e.g., A needs C > v1.0.0, B needs C > v1.1.0). Happy to contribute if we can find a path.
